sendgrid-php.php

```diff
@@ -1,3 +1,28 @@
 <?php
-require __DIR__ . '/vendor/autoload.php';
-?>
\ No newline at end of file
+/**
```

Comment:
Please add tests for this file.

Reply:
Testing functionality of it is difficult. This is because Composer is already loaded, and class `SendGrid\Mail\Mail` exists at that time.
Adding an `assertFileExists` in `FilesExistTest` is the only scenario which gets close to a - so called - 'test'.